dom-element.js

```diff
@@ -128,6 +128,10 @@ DOMElement.prototype.getAttributeNS =
 
 DOMElement.prototype.removeAttributeNS =
     function _Element_removeAttributeNS(namespace, name) {
+        var forbiddenKeys = ['__proto__', 'constructor', 'prototype'];
+        if (forbiddenKeys.includes(name)) {
```

Comment:
How would `name` being any of these keys lead to a prototype pollution?

`name` is only used in `delete attributes[name]` below. 
I don't think you can pollute the prototype using `delete` and you can't delete the prototype or constructor either.